scripts/source-maintenance.sh

```diff
@@ -108,6 +108,9 @@ if ! git diff --quiet; then
 	exit 1
 fi
 
+made="generated" # a hack: prevents $GeneratedByMe searches matching this file
+GeneratedByMe="This file is $made by scripts/source-maintenance.sh."
```

Comment:
Many other such disclaimers do not mention what will happen to edits and, more importantly, I failed to find a good way to describe what will happen to edits. All my attempts resulted in statements that were either useless because they were obvious or technically inaccurate because this file is not regenerated in most common circumstances (so editing it in some special cases will "work" and might even be useful).